packages/app-store/closecomothercalendar/test/lib/CalendarService.test.ts

```diff
@@ -225,7 +225,7 @@ test("prepare data to create custom activity type instance: one attendees, with
     attendees,
     startTime: now.toISOString(),
     additionalNotes: "Some comment!",
-  } as CalendarEvent;
+  } as any;
```

Comment:
Even though this is similar to line 171 it was still failing. Passes when cast as any type. This variable is declared anyways and does not rely on any external calls